### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by ayan-usgs**
* Corrected spelling (and import) of NotImplementedError
* replaced call to point_in_poly with point_in_tri call.

### CODE REVIEW

**Comment by daf** [comment]
Why does it think every line in the files have changed?  Line-endings getting switched around?  (based on OS/editor etc).  Is there a way to make this less noisy?

File: .settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (ChrisBarker-NOAA) [inline .settings/org.eclipse.core.resources.prefs]: Do we want Eclipse preferences in the repo? I"d rather not, but I don't expect they'd get in the way.

**Comment by ChrisBarker-NOAA** [comment]
IS that what the "Normalize Line endings" commit is supposed to fix? If so, it's be better to have that as a single pull requests, because as it is, I have no idea what's changed.

(but I sure wish gitHub's diff (or is it git's diff?) could ignore line endings!

**Comment by ocefpaf** [comment]
> but I sure wish gitHub's diff (or is it git's diff?) could ignore line endings!

It does.  The dark green and dark pink show what changed.  However, this would be more appropriated:

https://help.github.com/articles/dealing-with-line-endings/

**Comment by ayan-usgs** [comment]
@ChrisBarker-NOAA ,

I'm going to say, no you don't want that eclipse file as a part of repository. I've been fussing with https://help.github.com/articles/dealing-with-line-endings/ to no avail for the past half hour to remove the "noise".

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
```
Comment (ChrisBarker-NOAA) [inline .gitattributes]: Let's at least add *.py here ;-)

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# Declare files that will always have lf line endings on checkout.
+*.sln text eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```
Comment (ChrisBarker-NOAA) [inline .gitattributes]: and we'd better add *.nc here

Though auto does a pretty good job

**Comment by ayan-usgs** [comment]
Probably as "quiet" as it's going to get.

**Comment by ChrisBarker-NOAA** [comment]
OK -- were the line endings in make.bat DOS line endings? which his probably right, but if they've been switched to *nix-stle, they should get auto-fixed when cloned.

What's up with changing the permission on the scripts? It looks like have all been made non-executable.

**Comment by QuLogic** [comment]
I'm not sure if the line ending changes are necessary (I'd just avoid Notepad), but you can tack [?w=1 on to the diff](https://github.com/pyugrid/pyugrid/pull/57/files?w=1) to get something easier to read.

**Comment by ayan-usgs** [comment]
It would seem that Windows is not the best at figuring out Linux file permissions (http://blog.lesc.se/2011/11/how-to-change-file-premissions-in-git.html). The changes of filemode from 755 to 644 was not intended; they have been returned to 755.

**Comment by rsignell-usgs** [comment]
@ayan-usgs, what is the status of this PR.  Still seems to have an issue with line endings. 
Should we close it and you can try again?
For small changes you can use the editor on GitHub and it will fork/branch for you.

**Comment by ChrisBarker-NOAA** [comment]
someone should check if the sphinx bat file still works on Windows -- otherwise, looks good.

**Pull request merged**